sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -56,21 +54,66 @@
       :data="initialData"
       :readOnly="submitting"
       @submitted="submitted"
+      @loaded="formLoaded"
     ></formio>
+    <v-row v-if="showNav">
+      <v-col>
+        <v-btn
+          color="primary"
+          v-show="!isFirstPage"
+          variant="outlined"
+          data-cy="previousSection"
+          @click="wizardGoPrevious"
+          >Previous section</v-btn
+        >
+      </v-col>
+      <v-col>
+        <v-btn
+          class="float-right"
+          color="primary"
+          v-show="!isLastPage"
+          @click="wizardGoNext"
+          >Next section</v-btn
+        >
+        <v-btn
+          class="float-right"
+          :disabled="!isLastPage || submitting"
+          v-show="!isFirstPage"
+          color="primary"
+          @click="wizardSubmit()"
+        >
+          <v-progress-circular
+            v-if="submitting"
+            class="mr-3"
+            bg-color="white"
+            indeterminate
+            color="secondary"
+            size="23"
+          />
+          {{ submitting ? "Submitting..." : "Submit application" }}
```

Comment:
mm.. the label is 
![image](https://user-images.githubusercontent.com/77353155/196773884-defa03ac-da37-4630-b371-a67410f23955.png)

Reply:
Ok. Done. Thanks!